tests/src/baseservices/exceptions/regressions/Dev11/147911/fpcw.cpp

```diff
@@ -23,6 +23,8 @@ LONG WINAPI ExcepFilter(struct _EXCEPTION_POINTERS *pExp)
 
 
 // This is an example of an exported function.
+// Disable optimization otherwise compiler removes dividebyzero operation
```

Comment:
I have verified in disassembly that for release builds the divide by zero operation does not happen.